test/HostActivationTests/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "1.0.0-*",
+
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc3-004338"
```

Comment:
discussed offline, kept to an older version to keep VS and cmd line compat, no loss in test coverage since we explicitly test dotnet-test-xunit scenarios.